composer.json

```diff
@@ -59,7 +59,11 @@
         "vimeo/psalm": "^4.7"
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
```

Comment:
this is unrelated change, it needs comment from you

Reply:
It's a new behavior of Composer. This is added when running `composer update`.